core/internal/gethwrappers/generation/compile_command.txt

```diff
@@ -1 +1 @@
-yarn workspace @chainlink/contracts belt compile solc
+yarn workspace @chainlink/contracts compile:native
```

Comment:
Again, why go through yarn? Why not just call the script directly? Also can we just replace references from this file to `./scripts/native_solc_compile_all`?